main.py

```diff
@@ -1,16 +1,22 @@
-# This is a sample Python script.
+from selenium import webdriver
```

Comment:
You can remove this file from your project

Reply:
I have deleted this file